website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -33,14 +33,16 @@ Envoy must be run with the `--max-obj-name-len` option set to `256` or greater f
 
 The following matrix describes Envoy compatibility for the currently supported **n-2 major Consul releases**. For previous Consul version compatibility please view the respective versioned docs for this page.
 
+~> **Note:** Starting on Consul v1.14, Consul dataplane was implemented to manage Envoy proxies without the use of Consul clients. This means that compatibile Envoy versions for 1.14+ will be automatically upgraded via the consul-dataplane binary. 
```

Comment:
I think it would be helpful to have a note before providing the supported versions so readers understand why Consul v1.15 and v1.14 show Envoy 1.25.x and 1.24.x as supported(consul dataplane).

Reply:
My understanding is that this is only true in environments where Consul dataplane can currently be used (service mesh on Kubernetes), not in all cases (service mesh on non-k8s runtimes).

Reply:
Thanks for clarifying this @jkirschner-hashicorp 🙏